src/elm/Session/LoggedIn.elm

```diff
@@ -870,8 +911,25 @@ update msg model =
             { model | showCommunitySelector = False }
                 |> UR.init
 
-        SelectCommunity communityId ->
-            { model | selectedCommunity = communityId, showCommunitySelector = False }
+        SelectCommunity communityId shop actions ->
+            { model
+                | selectedCommunity = communityId
+                , showCommunitySelector = False
```

Comment:
just making more clear why I think features shouldn't be on this `Msg`. Here we simply set a new value and control visual stuff `showCommunitySelector`.

But then features stuff is added to another field from the old community selected. setting the features should have its own cmd and callback to be sure that we have the community stuff available when setting them. From just this you have no way of making sure we don't use the wrong community features